mayastor-macros/actix/src/lib.rs

```diff
@@ -18,7 +18,9 @@ impl Method {
         if attr.len() < 2 {
```

Comment:
You are doing another attr.remove() below, is this 2 still valid?

Reply:
good catch, should be 3 :+1: